docs/intro/from-ground-up.rst

```diff
@@ -215,41 +232,15 @@ to_item() method
 It is common to have Page Objects for a web page where a single main
```

Comment:
Should we also move this section much earlier? since the `to_item()` method has been already defined multiple times up top.

Reply:
it makes sense to add a note earlier; added it in https://github.com/scrapinghub/web-poet/pull/74/commits/28f917a3b7913c69ce3774e7d1d55105ebc6ff14. 

I'm not sure we can move the whole section earlier though. Currently the flow is this:

1. Show how code can be refactored without web-poet
2. Replicate the same thing with web-poet ("The same, but using web-poet")
3. Explain various concepts, starting from to_item.

We can't switch the sections, because web-poet is just introduced in a previous section. 
Maybe it's possible to write it all in some different way, but I'm out of ideas now :)

Reply:
I think that's fine. What do you think about adding a link pointing to the to_item() section instead to lead the read to more info? Though I think this would be a bit unnecessary since it would already be in the next section.

Reply:
I think it might be unnecessary; users should better keep reading, not jumping ahead :)